GPU/GPUTracking/CMakeLists.txt

```diff
@@ -14,6 +14,20 @@ set(MODULE GPUTracking)
 # set(CMAKE_CXX_FLAGS_${CMAKE_BUILD_TYPE_UPPER} "${CMAKE_CXX_FLAGS_${CMAKE_BUILD_TYPE_UPPER}} -O0") # to uncomment if needed, tired of typing this...
 # set(GPUCA_BUILD_DEBUG 1)
 
+# Pass ORT variables as a preprocessor definition
```

Comment:
Please do not use env variables here. These should be CMake variables. And then you should pass them to CMake in alidist/o2.sh

Reply:
Also, please make the compile definitions private, so they do not leak to dependent libraries.